gdk4/src/subclass/paintable.rs

```diff
@@ -7,88 +7,105 @@ use glib::Cast;
 
 pub trait PaintableImpl: ObjectImpl {
     fn get_current_image(&self, paintable: &Self::Type) -> Paintable {
-        unsafe {
-            let type_ = ffi::gdk_paintable_get_type();
-            let iface = glib::gobject_ffi::g_type_default_interface_ref(type_)
-                as *mut ffi::GdkPaintableInterface;
-            assert!(!iface.is_null());
+        self.parent_get_current_image(paintable)
```

Comment:
Are there default impls provided for all these in C? Only provide default impls in the trait for the functions where that is the case :) Generally for interface `Impl` traits

Reply:
Yes, there are defaults for all those functions